libs/product/src/drivers/magento/models/product-node.ts

```diff
@@ -9,9 +6,29 @@ export interface ProductNode {
   name: string;
   sku: string;
   url_key: string;
-  image: ProductImageNode;
-  price: ProductPriceNode;
-  media_gallery_entries?: any;
-  short_description?: any;
-  description?: any;
+  image: {
+		url: string,
+		label: string
+	};
+  price: {
+		regularPrice: {
+			amount: {
+				value: number
+				currency: any
+			}
+		}
+	};
+  media_gallery_entries?: {
+		label: string,
+		file: string,
+		position: number,
+		disabled: boolean,
+		id: number
+	}[];
+  short_description?: {
+		html: string;
+	};
+  description?: {
+		html: string;
+	};
```

Comment:
I think eventually we will want these extracted to their own "core" models but that can be done later.

Reply:
I thought about doing that, but it seems like it's difficult to keep track of what all the return values should be unless they're all in the same place. It's a lot easier to go back and forth between magento documentation and this, than between magento documentation and a bunch of objects. That was my thinking anyway

Reply:
At least for models such as image or money, I think many different models will end up using it and that would make it worth the additional complexity of having multiple files. We definitely do have to consider exactly what fields we need from the response so all of these might not get extracted. Either way, its something that can be better addressed later.